bgpd/bgp_vty.c

```diff
@@ -12244,6 +12244,7 @@ lcommunity_list_set_vty (struct vty *vty, int argc, struct cmd_token **argv,
       return CMD_WARNING;
     }
 
+  idx = 0;
```

Comment:
```
    argv_find (argv, argc, "AA:BB:CC", &idx);
    argv_find (argv, argc, "LINE", &idx);
    /* Concat community string argument. */
   if (idx)
     str = argv_concat (argv, argc, idx);
   else
     str = NULL;
```

the problem is with the zero check, idx will never be 0 here so a command like `ip large-community-list 22 deny` results in `22 deny` being passed as the community identifier